crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -104,6 +105,7 @@ <h2 class="h1">{{ current_step_title }}</h2>
     </div>
   </div>
 {% include "partials/unsupported-browser-modal.html" %}
+{% include "partials/contact-info-confirmation-modal.html" %}
```

Comment:
This is also in report_contact_info.html, and I think that is the better place for it.

Reply:
That makes sense!